### PR TITLE
Remove help properties file in roxctl

### PR DESCRIPTION
**Pull request by dhaus67**
## Description

This PR removes the `help.properties` file which previously held all the command's `Short/Long` descriptions.

The motivation to remove this are the following:
- multi-line text and more complex help is not as easy to define within the properties file as in-line.
- the properties file as abstraction currently doesn't provide any additional value. No translations or any other things are deduced from it.
- each command _required_ an entry within it, irrespective of whether it wanted to define a description or not, increasing the maintenance effort.

Instead of defining the descriptions within the properties file, they are now defined where the command is constructed using the cobra command's `Short/Long` property.

## Checklist
- [ ] Investigated and inspected CI test results
- [ ] Unit test and regression tests added
- [ ] Evaluated and added CHANGELOG entry if required
- [ ] Determined and documented upgrade steps
- [ ] Documented user facing changes (create PR based on [openshift/openshift-docs](https://github.com/openshift/openshift-docs) and merge into [rhacs-docs](https://github.com/openshift/openshift-docs/tree/rhacs-docs))

If any of these don't apply, please comment below.

## Testing Performed

- see CI passing.

### CODE REVIEW

**Comment by dhaus67** [comment]
Current dependencies on/for this PR:
* master
  * **PR #5447** <a href="https://app.graphite.dev/github/pr/stackrox/stackrox/5447" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/stackrox/stackrox/5447?utm_source=stack-comment).

File: roxctl/central/whoami/whoami.go [hunk 1]
```diff
@@ -24,7 +24,9 @@ type centralWhoAmICommand struct {
 // Command defines the central command tree
 func Command(cliEnvironment environment.Environment) *cobra.Command {
 	cbr := &cobra.Command{
-		Use: "whoami",
+		Use:   "whoami",
+		Short: "Display information about the current user and their authentication method.",
+		Long:  "Display information about the current user and their authentication method.",
```
Comment (janisz) [inline roxctl/central/whoami/whoami.go:29]: Long is same as short

File: roxctl/central/generate/k8s.go [hunk 2]
```diff
@@ -167,6 +169,8 @@ func openshift(cliEnvironment environment.Environment) *cobra.Command {
 		}
 		return clusterType, nil
 	})
+	c.Short = "Generate the required YAML configuration files to deploy StackRox Central into an OpenShift cluster."
+	c.Long = "Generate the required YAML configuration files to deploy StackRox Central into an OpenShift cluster."
```
Comment (janisz) [inline roxctl/central/generate/k8s.go:173]: Long is same as short. Let's keep short only

File: roxctl/image/check/check.go [hunk 3]
```diff
@@ -90,8 +90,10 @@ func Command(cliEnvironment environment.Environment) *cobra.Command {
 	pkgUtils.Must(err)
 
 	c := &cobra.Command{
-		Use:  "check",
-		Args: cobra.NoArgs,
+		Use:   "check",
+		Short: "Check images for build time policy violations, and report them.",
+		Long:  "Check images for build time policy violations, and report them.",
```
Comment (janisz) [inline roxctl/image/check/check.go:95]: Long is same as short. Let's keep short only

File: roxctl/maincommand/command.go [hunk 4]
```diff
@@ -26,8 +26,10 @@ import (
 
 func versionCommand(cliEnvironment environment.Environment) *cobra.Command {
 	c := &cobra.Command{
-		Use:  "version",
-		Args: cobra.NoArgs,
+		Use:   "version",
+		Short: "Display the current roxctl version.",
+		Long:  "Display the current roxctl version.",
```
Comment (janisz) [inline roxctl/maincommand/command.go:31]: Long is same as short. Let's keep short only

File: roxctl/scanner/generate/generate.go [hunk 5]
```diff
@@ -91,8 +91,10 @@ func Command(cliEnvironment environment.Environment) *cobra.Command {
 	scannerGenerateCmd := &scannerGenerateCommand{env: cliEnvironment}
 
 	c := &cobra.Command{
-		Use:  "generate",
-		Args: cobra.NoArgs,
+		Use:   "generate",
+		Short: "Generate the required YAML configuration files to deploy StackRox Scanner",
+		Long:  "Generate the required YAML configuration files to deploy StackRox Scanner",
```
Comment (janisz) [inline roxctl/scanner/generate/generate.go:96]: Long is same as short. Let's keep short only

File: roxctl/scanner/uploaddb/uploaddb.go [hunk 6]
```diff
@@ -59,8 +59,10 @@ func Command(cliEnvironment environment.Environment) *cobra.Command {
 	scannerUploadDbCmd := &scannerUploadDbCommand{env: cliEnvironment}
 
 	c := &cobra.Command{
-		Use:  "upload-db",
-		Args: cobra.NoArgs,
+		Use:   "upload-db",
+		Short: "Upload a vulnerability database for the StackRox Scanner.",
+		Long:  "Upload a vulnerability database for the StackRox Scanner.",
```
Comment (janisz) [inline roxctl/scanner/uploaddb/uploaddb.go:64]: Long is same as short. Let's keep short only

**Review by janisz — APPROVED** [review]

File: go.mod [hunk 7]
```diff
@@ -60,7 +60,6 @@ require (
 	github.com/joshdk/go-junit v1.0.0
 	github.com/lib/pq v1.10.7
 	github.com/machinebox/graphql v0.2.2
-	github.com/magiconair/properties v1.8.7
```
Comment (janisz) [inline go.mod:63]: ❤️

**Pull request merged**